packages/dom/src/dom.js

```diff
@@ -257,14 +257,16 @@ export function computeCaretRect() {
  *
  * @param {Element} container Focusable element.
  * @param {boolean} isReverse True for end, false for start.
+ * @param {Object} focusOptions options that get passed to the focus() call.
  */
-export function placeCaretAtHorizontalEdge( container, isReverse ) {
+export function placeCaretAtHorizontalEdge( container, isReverse, focusOptions = {} ) {
```

Comment:
`focusOptions` is really specific, and I don't think that's the right way to go. It implies that other devs will have to add more params to this function if they want to add other options that are unrelated to focus.

`placeCaretAtHorizontalEdge( container, isReverse, options )` is more scaleable for future changes to the function. I also don't think this object should be passed straight into `container.focus`, `preventScroll` is the only thing that function cares about, so it can be destructured from a more general options object.

Reply:
You are right :) of course.